packages/craftcms-webpack/index.js

```diff
@@ -343,7 +343,7 @@ const getConfig = ({
       config.plugins.push(new Dotenv({ path: dotenvResult }));
     }
 
-    if (!isDevServerRunning) {
+    if (!process.env.NODE_ENV === "production") {
       config.plugins.push(new CleanWebpackPlugin());
```

Comment:
Seems conceivable you'd want to do a build and not have it be minified. Since env is development when serving anyway, this allows for that flexibility.

Reply:
Yup, good point!